ipynb/energy/EnergyModel_ClusterEnergy.ipynb

```diff
@@ -106,12 +106,12 @@
     "    \"emeter\" : {\n",
     "        'instrument' : 'hwmon',\n",
     "        'conf' : {\n",
-    "            'sites' : [ 'a53', 'a57' ],\n",
+    "            'sites' : [ 'BOARDLITTLE', 'BOARDBIG' ],\n",
```

Comment:
This is the same sites naming used by HWMON on JunoR2 boards... is that a JunoR0 or a JunoR2 board?
Because, as @mdigiorgio previously commented, if you are using a JunoR2 board you should better set
```
"board" = "juno2" 
```
in the previous line 92, at that point you can avoid the "emeter" section here because is automatically specified for each supported platform.